docs/running-on-yarn.md

```diff
@@ -304,6 +304,14 @@ If you need a reference to the proper location to put log files in the YARN so t
   </td>
 </tr>
 <tr>
+  <td><code>spark.yarn.attemptFailuresValidityInterval</code></td>
+  <td>-1</td>
+  <td>
+  Ignore the failure number which happens out the validity interval (in millisecond).
```

Comment:
I think this configuration is quite similar to "spark.yarn.maxAttempts" which does not add "am" as scope constraint, do we also need to add "am" here in this new configuration?